src/Lean/Util/FoldConsts.lean

```diff
@@ -66,8 +66,26 @@ opaque foldConsts {α : Type} (e : Expr) (init : α) (f : Name → α → α) :
 def getUsedConstants (e : Expr) : Array Name :=
   e.foldConsts #[] fun c cs => cs.push c
 
+/-- Like `Expr.getUsedConstants`, but produce a `NameSet`. -/
+def getUsedConstants' (e : Expr) : NameSet :=
+  e.foldConsts {} fun c cs => cs.insert c
+
 end Expr
 
+namespace ConstantInfo
+
+/-- Return all names appearing in the type or value of a `ConstantInfo`. -/
+def getUsedConstants (c : ConstantInfo) : NameSet :=
```

Comment:
Just above, `getUsedConstants'` is the version that returns a `NameSet`, but down here it's the version without the `'` that does it. This seems likely to confuse someone in the future.

Reply:
Done.